CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +0,0 @@
-- Remove: RPM stuff (no longer used)
-- Remove: expressionLanguage field (as it it not longer needed, due to only one expression language is supported, after the removal of legacy expressions in iotagent-node-lib 3.2.0) (iota-node-lib#1384)
```

Comment:
To be moved to rpm/SPECS/iotamanager.spec, along with the entry about the upgrade in the library version.

Reply:
rpm/SPECS/iotamanager.spec was removed by https://github.com/telefonicaid/iotagent-manager/pull/311

Reply:
Use Changelog file for this

Reply:
fixed in https://github.com/telefonicaid/iotagent-manager/pull/313/commits/88bc7019d21a689b7b4f8137eb9d5226eba4b799